python_modules/libraries/dagster-dbt/dagster_dbt/cli/app.py

```diff
@@ -114,7 +114,7 @@ def copy_scaffold(
         if path.suffix == ".jinja":
             relative_path = path.relative_to(Path.cwd())
             destination_path = os.fspath(relative_path.parent.joinpath(relative_path.stem))
-            template_path = os.fspath(path.relative_to(dagster_project_dir))
+            template_path = path.relative_to(dagster_project_dir).as_posix()
```

Comment:
Would this not break on Windows?

Reply:
`.as_posix()` will create a forward-slash path, regardless of what the current OS is: https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.as_posix